app/src/main/java/org/p2p/wallet/newsend/analytics/NewSendAnalytics.kt

```diff
@@ -88,8 +93,69 @@ class NewSendAnalytics(
         isMaxButtonClicked = isClicked
     }
 
+    fun logSendQrGoingBack(
+        qrCameraIsAvailable: Boolean,
+        qrGalleryIsAvailable: Boolean,
+        qrTab: QrTab
+    ) {
+        analytics.logEvent(
+            NEW_SEND_QR_GOING_BACK,
+            mapOf(
+                "QR_Camera_Availability" to qrCameraIsAvailable,
+                "QR_Gallery_Availability" to qrGalleryIsAvailable,
+                "QR_Tab" to qrTab.title,
+            )
+        )
+    }
+
+    fun logSendActionButtonClicked() {
+        analytics.logEvent(NEW_SEND_ACTION_BUTTON)
+    }
+
+    fun logSendShowingDetails(
+        sendStatus: SendStatus,
+        lastScreenName: String,
+        tokenName: String,
+        sendNetwork: AnalyticsSendNetwork,
+        sendSum: BigDecimal,
+        sendUSD: BigDecimal
+    ) {
+        analytics.logEvent(
+            NEW_SEND_SHOWING_DETAILS,
+            mapOf(
+                "Send_Status" to sendStatus.title,
+                "Last_Screen" to lastScreenName,
+                "Token_Name" to tokenName,
+                "Send_Network" to sendNetwork.title,
+                "Send_Sum" to sendSum,
+                "Send_USD" to sendUSD
+            )
+        )
+    }
+
     enum class RecipientSelectionType(val type: String) {
         RECENT("Recent"),
         SEARCH("Search")
     }
+
+    enum class AnalyticsSendNetwork(val title: String) {
+        SOLANA("Solana"),
+        BITCOIN("Bitcoin")
+    }
+
+    private fun NetworkType.toAnalyticsValue(): AnalyticsSendNetwork = when (this) {
+        NetworkType.SOLANA -> AnalyticsSendNetwork.SOLANA
+        NetworkType.BITCOIN -> AnalyticsSendNetwork.BITCOIN
+    }
+
+    enum class SendStatus(val title: String) {
```

Comment:
i suggest adding Analytic in the name to remove ambiguity when using autocomplete / global search

Reply:
will do in a separate PR